src/main/java/com/philderbeast/prisonpicks/Config.java

```diff
@@ -48,19 +42,26 @@
     static int DIAMOND_PRIORITY = 7;
     static int DIAMOND_BLOCK_PRIORITY = 8;
     static int EMERALD_PRIORITY = 9;
+    static int COPPER_PRIORITY = 10;
```

Comment:
why are these priorities higher then Emeralds?

noting emeralds are the gambling currency they should probably still be the highest. and the ordering on the rest probably needs to be updated into order of value (unless the copper is higher then the value of a diamond block.

Reply:
planned on adding copper to a special mine

Reply:
and the rest of the materials? are they really more valuable then emeralds?